### ISSUE TITLE
[Question] How to recreate SEED benchmark results?

### ISSUE THREAD

**Issue by shakedpe** [haotian-liu/LLaVA#964]
### Question

Im looking at the script under "scripts/v1_5/eval/seed.sh" and the is no "./playground/data/eval/seed_bench" data to use in order to run script. Also when looking at the code you run "model_vqa_loader.py" is it unclear when you generate an answer how you use the questions +answers to get a final A/B/C/D answer in order to get you accuracy results. Can you please explain how to recreate the results? 
Thanks.

**Comment by simon-lund** [comment]
Hello, 

you have to download the Seed Bench data yourself (seed-bench v1). Instructions on how to set up the different benchmarks for evaluation can be found here: https://github.com/haotian-liu/LLaVA/blob/main/docs/Evaluation.md

Once you have set up the evaluation, you will find a file "llava-seed-bench.jsonl" under ./playground/data/eval/seed_bench". This file contains the questions as well as the possible answers. Thus, the model only  has to choose the letter corresponding to the correct answer as instructed.

Example:
```json
{
  "image": "SEED-Bench-image/1454426_2591111986", 
  "text": "How many towels are in the image?\nA. One\nB. Two\nC. Three\nD. Four\nAnswer with the option's letter from the given choices directly.", 
  "category": "Instances Counting", 
  "question_id": 101669}
```
I hope this helps.

**Comment by NimrodShabtay** [comment]
Hi, @simon-lund 
Is there an easy way to evaluate just the image questions' types?
Thanks

**Comment by simon-lund** [comment]
Sorry, I don't know.

It's been a while since I last used it.

**Comment by HuangChiEn** [comment]
> Hi, [@simon-lund](https://github.com/simon-lund) Is there an easy way to evaluate just the image questions' types? Thanks

yes, the script will automatically skip the video part, if your video image doesn't properly placed at the indicated location. So, you just need to prepare the SEED image part then run the script, it will give you the image part results.
(but a simple hot-fix for py script is needed)

![Image](https://github.com/user-attachments/assets/d3095157-cca4-4194-9b9a-c0d949803123)
